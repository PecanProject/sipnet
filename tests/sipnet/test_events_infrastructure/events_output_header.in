0  2024  65  plant  3 4 5 6
0  2024  70  irrig  5 1
0  2024  200 harv   0.4 0.3 0.2 0.1
1  2024  65  plant  3 5 7 9
1  2024  70  irrig  5 0
1  2024  200 harv   0.1 0.1 0.1 0.1
