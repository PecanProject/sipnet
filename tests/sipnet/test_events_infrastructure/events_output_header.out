loc  year  day  type     param_name=delta[,param_name=delta,...]
0    2024   65  plant    envi.plantLeafC=3.00,envi.plantWoodC=4.00,envi.fineRootC=5.00,envi.coarseRootC=6.00
0    2024   70  irrig    envi.soilWater=5.00,fluxes.immedEvap=0.00
0    2024  200  harv     env.litter=5.46,envi.plantLeafC=-5.93,envi.plantWoodC=-4.75,envi.fineRootC=-3.73,envi.coarseRootC=-3.89
1    2024   65  plant    envi.plantLeafC=3.00,envi.plantWoodC=5.00,envi.fineRootC=7.00,envi.coarseRootC=9.00
1    2024   70  irrig    envi.soilWater=2.50,fluxes.immedEvap=2.50
1    2024  200  harv     env.litter=4.25,envi.plantLeafC=-1.39,envi.plantWoodC=-1.63,envi.fineRootC=-2.52,envi.coarseRootC=-2.97
